.ci/setup_profiles.sh

```diff
@@ -8,10 +8,10 @@ if [[ "$TEST_TYPE" != "pre-commit" ]]
     # no setup at all required for pre-commit to run
 then
     # Here I create the actual DB for submission
-    psql -h localhost -c "CREATE DATABASE $TEST_AIIDA_BACKEND;" -U postgres -w
+    psql -h localhost -c "CREATE DATABASE $TEST_AIIDA_BACKEND ENCODING \"UTF8\" LC_COLLATE=\"en_US.UTF-8\" LC_CTYPE=\"en_US.UTF-8\" TEMPLATE=template0;" -U postgres -w
```

Comment:
I fail to find the definition of ``template0`` database. Is it also created using utf-8 encoding? This thing might be inherited.

Reply:
@giovannipizzi told me that this is a built-in in postgres.

Reply:
This is ok and actually the suggested way of doing it, as explained in this documentation page:
https://www.postgresql.org/docs/current/static/sql-createdatabase.html
> The encoding and locale settings must match those of the template database, except when template0 is used as template. This is because other databases might contain data that does not match the specified encoding, or might contain indexes whose sort ordering is affected by LC_COLLATE and LC_CTYPE. Copying such data would result in a database that is corrupt according to the new settings. template0, however, is known to not contain any data or indexes that would be affected.